Bottom: a3c87b0a8e8b976fc6713dc9ef9e68b513fa6cbe
Top:    70bf4f0e3fc7280ac63a2fbe71ecc97d4858334a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-15 22:35:12 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 9b775ac..d1b0452 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -31,7 +31,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 static int step_set_layer (const char *name, int group, int empty);
 static void use_gc (hidGC gc);
